### PR TITLE
Separated virt-scenario from ALP guide

### PR DESCRIPTION
**Pull request by tbazant**
### Description

An attempt to make an article from ALP guide chapter

### CODE REVIEW

File: tasks/virt-scenario-create-vm.xml [hunk 1]
```diff
@@ -15,91 +15,84 @@
...
+      The <option>force_sev</option> option forces the extraction of the
+      Platform Diffie-Hellman key (PDH) on the current &amdsev; system. The PDH
+      file is used to negotiate a master secret between the SEV firmware and
+      the external entities. This file must be stored in a secure place, and
+      this option is only provided for <emphasis>testing</emphasis> purposes.
     </para>
+  </warning>
+  <para>
+    After you finish optional configuration steps, enter the name of one of the
+    scenarios&mdash;<literal>computation</literal>, <literal>desktop</literal>,
```
Comment (dariavladykina) [inline tasks/virt-scenario-create-vm.xml:57]:
```suggestion
    scenarios&mdash;<literal>computation</literal>, <literal>desktop</literal>
```

File: tasks/virt-scenario-create-vm.xml [hunk 2]
```diff
@@ -198,6 +191,13 @@ host_filesystem:
       or display its status by running
     </para>
 <screen>virt-scenario-launch --status <replaceable>VM_NAME</replaceable></screen>
+    <para>
+      For example, the <literal>securevm</literal> VM confirms SEV attestation:
```
Comment (dariavladykina) [inline tasks/virt-scenario-create-vm.xml:195]:
```suggestion
      For example, the <literal>securevm</literal> VM confirms the SEV attestation:
```

File: tasks/virt-scenario-create-vm.xml [hunk 3]
```diff
@@ -198,6 +191,13 @@ host_filesystem:
...
+    </para>
+<screen>&prompt.root;<command>virt-scenario-launch --start ALPOS</command>
+  Connected to libvirtd socket; Version: 7001000
+  SEV(-ES) attestation passed!
+  Validation successfull for domain ALPOS</screen>
     <para>
       For a complete list of the command's option, run
```
Comment (dariavladykina) [inline tasks/virt-scenario-create-vm.xml:202]:
```suggestion
      For a complete list of the command's options, run
```

File: references/virt-scenarios.xml [hunk 4]
```diff
@@ -28,7 +28,7 @@
   </info>
   <variablelist>
     <varlistentry>
-      <term>Secure VM</term>
+      <term>securevm</term>
       <listitem>
         <para>
           Selecting this scenario results in an encrypted VM image with high
```
Comment (dariavladykina) [inline references/virt-scenarios.xml:34]:
```suggestion
          Selecting this scenario results in an encrypted VM image with a high
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please consider some suggestions here. Thanks!

File: articles/virt-scenario-creating-customized-vm-guests.asm.xml [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+          <dm:component>Smart Docs</dm:component>
+          <dm:product>Documentation</dm:product>
+          <dm:assignee>tbazant@suse.com</dm:assignee>
+        </dm:bugtracker>
+        <dm:translation>yes</dm:translation>
+      </dm:docmanager>
+      <abstract>
+        <variablelist>
+          <varlistentry>
+            <term>TOPIC</term>
```
[the remark this answers is not in the document]
Comment (tbazant) [reply articles/virt-scenario-creating-customized-vm-guests.asm.xml:83]: @dariavladykina i took Frank's article as an example - https://susedoc.github.io/doc-modular/main/html/systemd-working-with-timers/

[the remark this answers is not in the document]
Comment (dariavladykina) [reply articles/virt-scenario-creating-customized-vm-guests.asm.xml:83]: That was a pilot article. Afterwards, we have confirmed that the What/why headings should be used.

**Review by lvicoun — APPROVED** [review]

File: references/virt-scenario-manage-vm.xml [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+   xmlns:xi="http://www.w3.org/2001/XInclude"
+   xmlns:xlink="http://www.w3.org/1999/xlink">
+  <info>
+    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
+    <title>Managing virtual machines with &virt-scenario;</title>
+  </info>
+  <para>
+    After you created a VM using the &virt-scenario; interactive shell, use the
+    <command>virt-scenario-launch</command> command to manage it. The command
+    identifies VMs by their name as displayed by the <option>--list</option>.
```
Comment (dariavladykina) [inline references/virt-scenario-manage-vm.xml:23]:
```suggestion
    identifies VMs by their name as displayed by the <option>--list</option> option.
```

File: references/virt-scenario-manage-vm.xml [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  </para>
+<screen>&prompt.root;<command>virt-scenario-launch --list</command>
+  Version: 2.1.2
+  Available VMs:
+    ALP_OS
+    desktop
+    testing_vm
+    SLE15_HPC</screen>
+  <para>
+    After you have identified the VM, use the following options to manage it.
```
[the remark this answers is not in the document]
Comment (dariavladykina) [reply references/virt-scenario-manage-vm.xml:33]: Just rewording here to avoid identical sentences (see the para above).

**Review by dariavladykina — COMMENTED** [review]
Some nitpicks :-)

**Pull request merged**